lib/config/presets/internal/default.ts

```diff
@@ -190,7 +190,10 @@ export const presets: Record<string, Preset> = {
   },
   docker: {
     description: 'Keep Dockerfile `FROM` sources updated.',
-    docker: {
+    'docker-compose': {
```

Comment:
didn't this match more managers before? 🤔 should we use `matchDatasource=docker` instead? 🤔

Reply:
Yes, but I think this shouldn't be the case as the preset explicitly mentions the `FROM` command. `matchDatasource=docker` would also include managers like `docker` or `helm`. These are all managers which had the Docker language attached. 

```typescript
const dockerLanguageManagers = [
  'ansible',
  'dockerfile',
  'docker-compose',
  'droneci',
  'kubernetes',
  'woodpecker',
];
```
https://github.com/renovatebot/renovate/pull/16534/files#diff-ee5c5f278fb802425d5b5731297b89c72bc71c0c0a562e6412fcaf415ad264e3R27-R34